test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
+      triggerEvent node, 'submit', data
 
       assert.isTrue context.fn.calledOnce
-      assert.isTrue context.fn.calledWith(data)
+      assert.isTrue context.fn.calledWith(data), 'fn called with data'
```

Comment:
Maybe try a debugger to look at `context.fn.args` (if I remember my sinon/chai properly) to see what it was indeed called with?  I'm guessing no args